### ISSUE TITLE
Expandable Cards - Accessibility Improvements

### ISSUE THREAD

**Issue by GCHQ-Developer-299** [mi6/ic-ui-kit#1520]
## Summary of the bug 

The Expandable Card variant has a couple of accessibility issues.
- [x] add aria-expanded=false to Expandable button on cards when expanded content is not visible
- [x] make the tooltip on the Expandable button silent, so screenreaders don't hear it read out twice
- [x] Ensure that the cypress issue identified in #1457 is fixed (this might be because ic-button has a shadow dom so aria-controls cannot be accessed. Maybe replace this button with a native <button>?

## 📸 Screenshots or code 
See below, no 'collapsed' and the label being read out twice:
![Screenshot 2024-02-15 at 15 28 03](https://github.com/mi6/ic-ui-kit/assets/111292477/0a64dea7-a2b5-4769-bb31-48fa4f51e318)

## 🖥 📱 Device 
- Type: [e.g. Desktop, mobile] 
- Device: [e.g. iPhone, MacBook, ThinkPad] 
- OS version: [e.g. macOS13, iOS16, Android 13] 
- Browser version: [e.g. Chrome 115, Safari 16] 

## 🧐 Expected behaviour 
Expect the button to tell screenreaders that it is collapsed when the expandable content is hidden, and expanded when the content is visible. The button should only read it's label out once.

## Additional info 
This was the outcome of a screenreader testing session.

**Comment by GCHQ-Developer-530** [comment]
This ticket might link to #1457
